pages/pigx-sars-cov2-ww.md

```diff
@@ -267,6 +271,31 @@ characterising variants of concern (VOC) of SARS-CoV-2 provided by
 [outbreak.info](https://outbreak.info/situation-reports) and
 [CoVariant.org](https://covariants.org/variants/S.501Y.V1).
 
+## Output Folder Structure
+
+The pipeline will create a specific directory structure in the, the respective contents are explained below:
+
+```
+|-- coverage
+|-- kraken
+|-- logs
+|-- mapped_reads
+|-- pigx_work
+|-- report
+|-- trimmed_reads
+|-- variants
+```
+
+| Folder    | description |
+|---------|-------------|
+|   coverage     | CSV files of amplicons covered and read coverage |
+|    kraken    | taxonomic classification files with NCBI taxonomy ID for unaligned reads generated with Kraken2 |
+|    logs    | detailed output from execution of each step of the pipeline |
+|    mapped_reads    | BAM and SAM files for aligned and unaligned reads against SARS-CoV-2 |
+|     pigx_work   | pigx related scripts  |
+|     report   | html files for reports descibed [here](#output-description)|
+|   trimmed_reads     | fastq files for trimmed reads |
+|   variants     | detected single nucleotide variants (SNVs) from the aligned reads in VCF/ CSV files and VEP output files as TXT/HTML |
```

Comment:
Please align the table so that it looks correct in ASCII.

"Folder" -> "directory"
"pigx" -> "PiGx"
"html" -> "HTML"
"TXT" -> "plain text"

The left column duplicates the list above, so please consider removing the earlier list.